### PR TITLE
docs(weekly): add this week in databend 118

### PR DESCRIPTION
**Pull request by PsiACE**
Almost forgot, and cc @soyeric128

### CODE REVIEW

File: blog/2023-11-05-databend-weekly-118.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### MERGE INTO Now Supports Distributed Execution
+
+The `MERGE INTO` statement performs insert, update, or delete operations on rows in the target table based on specified conditions and matching criteria using data from the specified source.
+
+In Databend, the `MERGE INTO` statement now supports distributed execution, which enables more efficient data integration capabilities. The physical plan for distributed execution is shown in the following diagram:
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:51]:
```suggestion
MERGE INTO has been enhanced to support distributed execution. The physical plan for distributed execution is shown as follows.
```

File: blog/2023-11-05-databend-weekly-118.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### MERGE INTO Now Supports Distributed Execution
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:47]:
```suggestion
### MERGE INTO now Supports Distributed Execution
```

File: blog/2023-11-05-databend-weekly-118.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+If you are interested in learning more, please check out the resources below:
+
+- [Blog | Incorporate Data Version Control into Your Analytics Workflow](https://databend.rs/blog/2023-11-02-databend-with-lakefs)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added support for JSON operators: `?`, `?|`, and `?&`.
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:79]:
```suggestion
- Introduced new [JSON operators](https://databend.rs/doc/sql-commands/query-operators/json/): `?`, `?|`, and `?&`.
```

File: blog/2023-11-05-databend-weekly-118.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- Added support for permissions management on `STAGE` and `UDF`.
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:80]: Could you share the PR link?

File: blog/2023-11-05-databend-weekly-118.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- Added support for `flatten` table function.
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:81]:
```suggestion
- Introduced new table function [FLATTEN](https://databend.rs/doc/sql-functions/semi-structured-functions/flatten).
```

File: blog/2023-11-05-databend-weekly-118.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+- Read the documentation [Example 2: Attaching Table in READ_ONLY Mode](https://databend.rs/doc/sql-commands/ddl/table/attach-table#example-2-attaching-table-in-read_only-mode) to learn how to use `ATTACH TABLE` in read-only mode.
+- Read the documentation [Conversion Functions](https://databend.rs/doc/sql-functions/conversion-functions/) to learn about type conversion functions and rules in Databend.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Support for Reading Delta Lake
+
+Databend currently supports mounting and reading data from Iceberg tables through the Catalog, and the next planned target is Delta Lake.
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:92]: After successfully incorporating support for mounting and reading data from Iceberg tables through the Catalog, Databend is gearing up for adding compatibility with Delta Lake.

File: blog/2023-11-05-databend-weekly-118.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+[Issue #13429 | add read only support for Delta table ](https://github.com/datafuselabs/databend/issues/13429)
+
+Please let us know if you're interested in contributing to this feature, or pick up a good first issue at <https://link.databend.rs/i-m-feeling-lucky> to get started.
+
+## New Contributors
+
+We always open arms to everyone and can't wait to see how you'll help our community grow and thrive.
+
+* [@guojidan](https://github.com/guojidan) added support for more filter options in the SHOW statements, [#13510](https://github.com/datafuselabs/databend/pull/13510).
```
Comment (soyeric128) [inline blog/2023-11-05-databend-weekly-118.md:104]:
```suggestion
* [@guojidan](https://github.com/guojidan) added new filter parameters for the SHOW commands in [#13510](https://github.com/datafuselabs/databend/pull/13510).
```

File: blog/2023-11-05-databend-weekly-118.md (hunk 4 above)
Comment (PsiACE) [reply blog/2023-11-05-databend-weekly-118.md:80]: https://github.com/datafuselabs/databend/pull/13445

**Pull request merged**